php/WcaOauth.php

```diff
@@ -19,12 +21,27 @@ class WcaOauth
 
     public function __construct($options)
     {
+        $this->setOptions($options);
+        $this->checkRequiredOptionsSet();
```

Comment:
How do you feel about moving the call to checkRequiredOptionsSet() into the setOptions() method? I can't imagine we'd ever want to call setOptions without then calling checkRequiredOptionsSet